spdx/v2/common/package.go

```diff
@@ -73,12 +73,14 @@ func (o *Originator) UnmarshalJSON(data []byte) error {
 	originatorFields := strings.SplitN(originatorStr, ": ", 2)
 
 	if len(originatorFields) != 2 {
-		return fmt.Errorf("failed to parse Originator '%s'", originatorStr)
+		originatorFields = strings.SplitN(originatorStr, ":", 2)
+		if len(originatorFields) != 2 {
+			return fmt.Errorf("failed to parse Originator '%s'", originatorStr)
+		}
```

Comment:
could we change this to just instead of splitting on `": "`, just modify that to `":"` and `strings.TrimLeft(originatorFields[1], " \t")` below. this way we don't need to have a second split case.